src/BizHawk.Emulation.Common/Base Implementations/Bk2MnemonicLookup.cs

```diff
@@ -802,6 +802,122 @@ public static string LookupAxis(string button, string systemId)
 				["Next Drive"] = '}',
 				["Next Slot"] = '>',
 			},
+			[VSystemID.Raw.DOS] = new()
+			{
+				["Joystick Button 1"] = '1',
+				["Joystick Button 2"] = '2',
+				["Joystick Up"] = 'U',
+				["Joystick Down"] = 'D',
+				["Joystick Left"] = 'L',
+				["Joystick Right"] = 'R',
+				["Mouse Left Button"] = 'l',
+				["Mouse Middle Button"] = 'm',
+				["Mouse Right Button"] = 'r',
+				["Next Floppy Disk"] = '>',
+				["Next CDROM"] = '>',
```

Comment:
```suggestion
				["Next CDROM"] = '}',
```
It's better to use slightly different "arrows" for different operations, and cdrom is more advanced so it gets a curly brace :)